platforms/chibios/drivers/wear_leveling/wear_leveling_efl.c

```diff
@@ -15,8 +15,8 @@ static flash_sector_t first_sector = UINT16_MAX;
 #endif // defined(WEAR_LEVELING_EFL_FIRST_SECTOR)
 
 static flash_sector_t sector_count = UINT16_MAX;
-static BaseFlash *    flash;
-
+static BaseFlash     *flash;
```

Comment:
Please revert to appease lint.
```suggestion
static BaseFlash *    flash;
```

Reply:
funny enough, the auto formatter did that.